app/controllers/application_controller.rb

```diff
@@ -90,4 +90,14 @@ def set_visitor_props
       @visitor_properties = {user: current_user.email, role: current_user.user_role}
     end
   end
+
+  def log_spam_attempt(form)
+    ip_address = request.remote_ip
+    url_orig = request.original_url
+    email = params["email"]
+    subject = params["subject"]
+    message = params["message"]
+    phone = params["phone"]
+    FormSpam.create(form: form, original_url: url_orig, ip_address: ip_address, email: email, subject: subject, message: message, phone: phone)
+  end
```

Comment:
Can we move this to a module? I don't think this method is related to the `ApplicationController`. Also, I'm not sure how this is working because it's a private method.

Reply:
Yeah - I created a form_spams_helper to house the log_spam method and just included in the about and nominate controllers.  I did read a few articles that suggested just putting it in the application controller.. since multiple controllers need to access it.